invenio_pidstore/admin.py

```diff
@@ -24,25 +24,21 @@
 from flask import current_app, url_for
 from flask_admin.contrib.sqla import ModelView
 from flask_admin.contrib.sqla.filters import FilterEqual
+from flask_babelex import gettext as _
 from markupsafe import Markup
 
 from .models import PersistentIdentifier, PIDStatus
 
 
-def _(x):
-    """Identity function for string extraction."""
-    return x
-
-
 def object_formatter(v, c, m, p):
     """Format object view link."""
     endpoint = current_app.config['PIDSTORE_OBJECT_ENDPOINTS'].get(
         m.object_type)
 
     if endpoint and m.object_uuid:
         return Markup('<a href="{0}">{1}</a>'.format(
-                url_for(endpoint, id=m.object_uuid),
-                _('View')))
+            url_for(endpoint, id=m.object_uuid),
+            _('View')))
```

Comment:
Did you test this to see if it works? I previously had problems getting Flask-Admin to work with ``lazy_gettext`` strings….and using ``gettext`` here will not yield the correct result…i think